web-bluetooth/get-primary-services.js

```diff
@@ -0,0 +1,30 @@
+function onButtonClick() {
+  let filterName = document.querySelector('#name').value;
+
+  let services = document.querySelector('#services').value.split(/, ?/)
+    .map(s => s.startsWith('0x') ? parseInt(s) : s)
+    .filter(s => s && BluetoothUUID.getService);
+
+  log('Requesting Bluetooth Device...');
+  navigator.bluetooth.requestDevice({
+    filters: [{name: filterName}],
+    optionalServices: services
+  })
+  .then(device => {
+    log('Connecting to GATT Server...');
+    return device.gatt.connect();
+  })
+  .then(server => {
+    // Note that we could also get all services that match a specific UUID by
+    // simply passing it to getPrimaryServices().
+    log('Getting Services...');
+    return server.getPrimaryServices();
+  })
+  .then(services => {
+    log('> Services: ' +
+      services.map(s => s.uuid).join('\n' + ' '.repeat(19)));
```

Comment:
In that case, it should `12` not `19` spaces. Thanks!
See https://googlechrome.github.io/samples/web-bluetooth/get-characteristics.html for the alignment

![screenshot 2016-06-13 at 1 25 18 pm](https://cloud.githubusercontent.com/assets/634478/16005926/4990b6e6-316a-11e6-9692-83f893afa5ac.png)